src/main/webapp/WEB-INF/views/cars/driverToCar.jsp

```diff
@@ -0,0 +1,20 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: Tsven
+  Date: 21.01.2021
+  Time: 20:50
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Title</title>
+</head>
+<h1>Add driver to car page</h1>
+<form method="post" action="${pageContext.request.contextPath}driverToCar" >
+    Please, provide driver_id <input type="number" name="driver_id">
+    Please, provide car_id <input type="number" name="car_id">
+    <button type="submit">Register</button>
+</form>
+</table>
+</html>
```

Comment:
https://mate-academy.github.io/jv-program-fulltime/02_jdbc/checklist/08_servlets_practice_checklist.html
check urls naming for each url

Reply:
controller — singular,
controllers — plural;
So the package name should be controller

Reply:
Are you sure this? In video Bogdan called his directory controllers because there are a lot of different controllers.

Reply:
Argee with you...just followBogdan's video and there was plural naming for controllers.